tests/fixtures/httpApi/serverless.yml

```diff
@@ -1,6 +1,8 @@
 service: service
 provider:
   name: aws
+  httpApi:
```

Comment:
Updating it here, will affect also integration tests, and I assume will introduce a collision when tests are run concurrently.

Instead of hardcoding it here. Let's extend `httpApi` configuration just for one of the tests, as e.g. it's done here: https://github.com/serverless/serverless/blob/43633b2bc9c270e1d0589914d168a9b9d2f2d366/lib/plugins/aws/package/compile/events/httpApi/index.test.js#L201-L203